coordinator/src/routes.rs

```diff
@@ -542,3 +553,55 @@ pub async fn collaborative_revert_confirm(
     })?;
     Ok(Json(serialize_hex(&raw_tx)))
 }
+
+// TODO(holzeis): There is no reason the backup and restore api has to run on the coordinator. On
+// the contrary it would be much more reasonable to have the backup and restore api run separately.
+pub async fn backup(
+    Path(node_id): Path<String>,
+    State(state): State<Arc<AppState>>,
+    backup: Json<Backup>,
+) -> Result<(), AppError> {
+    let node_id = PublicKey::from_str(&node_id)
+        .map_err(|e| AppError::BadRequest(format!("Failed to backup {}. {e:#}", backup.key)))?;
+
+    state
+        .user_backup
+        .backup(node_id, backup.0)
+        .await
+        .map_err(|e| AppError::InternalServerError(e.to_string()))
+}
+
+// TODO(holzeis): There is no reason the backup and restore api has to run on the coordinator. On
+// the contrary it would be much more reasonable to have the backup and restore api run separately.
+pub async fn delete_backup(
```

Comment:
Under what circumstances do you expect `delete_backup` to be called?
Maybe we don't want to expose this endpoint?

Reply:
This is a necessity for `rust-dlc` as it is also deleting data. More specifically, `rust-dlc` deletes contracts with a temporary contract id, once the sub channel is established. I think we could keep that data, but opted to expose the api nonetheless to stay as close to the actual state as possible.